dolphinscheduler-server/src/main/resources/config/install_config.conf

```diff
@@ -76,19 +76,19 @@ apiServerPort="12345"
 # ---------------------------------------------------------
 # The type for the metadata database
 # Supported values: ``postgresql``, ``mysql``.
-dbtype="mysql"
+DATABASE_TYPE="mysql"
 
-# The <HOST>:<PORT> connection pair DolphinScheduler connect to the metadata database
-dbhost="192.168.xx.xx:3306"
+# Spring datasource driver-class-name
+SPRING_DATASOURCE_DRIVER_CLASS_NAME="com.mysql.jdbc.Driver"
```

Comment:
Let's remove this, because when `DATABASE_TYPE` is set, the driver class is set accordingly, and if we let the users to set these, they might set a wrong driver for the database type, example: set `com.mysql.jdbc.Driver` for PostgreSQL.

Reply:
Ok, I got your point, thanks~